tests/ci/submitpytest.py

```diff
@@ -0,0 +1,177 @@
+print("subscription_id ", subscription_id)
+
+cli_auth = AzureCliAuthentication()
+
+try:
+    print("Trying to get ws")
+    ws = Workspace.get(
+        name=workspace_name,
+        subscription_id=subscription_id,
+        resource_group=resource_group,
+        auth=cli_auth,
+    )
+
+except Exception:
+    # this call might take a minute or two.
+    print("Creating new workspace")
+    ws = Workspace.create(
+        name=workspace_name,
+        subscription_id=subscription_id,
+        resource_group=resource_group,
+        # create_resource_group=True,
+        location=location,
+        auth=cli_auth,
+    )
+
+# print Workspace details
+print(ws.name, ws.resource_group, ws.location, ws.subscription_id, sep="\n")
+#
+# https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/training/train-on-amlcompute/train-on-amlcompute.ipynb
+# useful experiment, etc info
+# https://docs.microsoft.com/en-us/python/api/overview/azure/ml/intro?view=azure-ml-py
+
+#
+# Create an Experiment
+#
+
+# experiment_name = 'train-on-amlcompute'
+experiment_name = 'unit_tests_staging'
+experiment = Experiment(workspace=ws, name=experiment_name)
+
+#
+# Check Available VM families
+#
+'''
+bz
+AmlCompute.supported_vmsizes(workspace=ws)
+# AmlCompute.supported_vmsizes(workspace = ws, location='southcentralus')
+
+#
+# Create a compute Resource
+# from here
+# https://docs.microsoft.com/en-us/azure/machine-learning/service/tutorial-train-models-with-aml
+# https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-set-up-training-targets
+#
+'''
+
+# create a new runconfig object
+run_config = RunConfiguration()
+
+# signal that you want to use AmlCompute to execute script.
+# set to 'local' instead of amlcompute when running in local docker
+run_config.target = "amlcompute"
+
+# AmlCompute will be created in the same region as workspace
+# Set vm size for AmlCompute
+# chosen from STANDARD_D2_V2 or STANDARD_NC6
+
+run_config.amlcompute.vm_size = 'STANDARD_D2_V2'
+
+
+# enable Docker
+run_config.environment.docker.enabled = True
+
+# set Docker base image to the default CPU-based image
+
+# run_config.environment.docker.base_image = DEFAULT_CPU_IMAGE
+# just to see if this is it bz
+run_config.environment.docker.base_image = DEFAULT_CPU_IMAGE
+
+# run_config.environment.docker.base_image = 'continuumio/miniconda3'
+
+# use conda_dependencies.yml to create a conda environment in the
+# Docker image for execution
+run_config.environment.python.user_managed_dependencies = False
+
+# auto-prepare the Docker image when used for execution (if it is not already
+# prepared)
+run_config.auto_prepare_environment = True
+
+print("reco_base.yaml path")
+print(os.path.dirname('./reco_base.yaml'))
+print('reco_base.yaml exists ', os.path.exists('./reco_base.yaml'))
+
+# specify CondaDependencies obj
+run_config.environment.python.conda_dependencies = CondaDependencies(
+    conda_dependencies_file_path='./reco_base.yaml')
+
```

Comment:
we may want this to be agnostic to the yaml file used (reco_base, reco_gpu, etc.) while we could name them all reco_base, I would suggest just changing this to reco.yaml, and in the generation script passing the name argument 'reco'.

Reply:
The proper yaml file is generated in each pipeline and agree that it's better to change to reco.yaml.  What if I do that in the next pass so I can test the first pass?

Reply:
done